pkg/scheduler/api/node_info.go

```diff
@@ -26,6 +26,7 @@ import (
 
 	"volcano.sh/apis/pkg/apis/scheduling/v1beta1"
 	"volcano.sh/volcano/pkg/scheduler/api/devices/nvidia/gpushare"
+	"volcano.sh/volcano/pkg/scheduler/api/devices/nvidia/vgpu4pd"
```

Comment:
please change the package name from `vgpu4pd` to `vgpu`

Reply:
fixed